src/System.Private.CoreLib/shared/System/Convert.Base64.Avx2.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
+
+namespace System
+{
+    public static partial class Convert
+    {
+        // Based on "Base64 encoding with SIMD instructions" article by Wojciech Muła
+        // http://0x80.pl/notesen/2016-01-12-sse-base64-encoding.html
+
+
+        // SSSE3-based implementation
+
+        private static unsafe void EncodeBase64Sse(byte* input, int inputLength, byte* output, int outputLength)
+        {
+            byte* outputCurrent = output;
+            int stride = 4 * 3;
+
+            Vector128<byte> shuffleMask = Vector128.Create((byte)1, 0, 2, 1, 4, 3, 5, 4, 7, 6, 8, 7, 10, 9, 11, 10);
+            Vector128<byte> shiftLut = Vector128.Create(18229723555195321415UL, 72503040736508UL).AsByte();
```

Comment:
these longs are Vector<ulong> representation of https://github.com/WojciechMula/base64simd/blob/master/encode/lookup.avx2.cpp#L72-L80
will fix it